puzzle-utils/words/wordsearch.py

```diff
@@ -105,10 +127,14 @@ def wordsearch_brute_function(word, row, column, direction, min_length):
         return WordSearchResult(word, row, column, direction)
 
 def wordsearch_grid(grid, function, parameters):
```

Comment:
While we're here, can you add like type annotations or some comments to be explicit about what to expect in these?